site/content/en/docs/concepts/admission_check.md

```diff
@@ -1,23 +1,22 @@
 ---
 title: "Admission Check"
-date: 2023-10-05
+date: 2024-13-06
 weight: 6
 description: >
-  A mechanism allowing internal or external components to influence the timing of
-  workloads admission.
+  Mechanism allowing internal or external components to influence the workload's admission.
 ---
 
 AdmissionChecks are a mechanism that allows Kueue to consider additional criteria before admitting a Workload.
-When a ClusterQueue has AdmissionChecks configured, each of the checks has to provide a
-positive signal to the Workload before it can be [Admitted](https://kueue.sigs.k8s.io/docs/concepts#admission).
+Kueue runs all AdmissionChecks at the same time, after a Workload has reserved the quota.
+All of the AdmissionChecks have to provide a positive signal to the Workload before it can be [Admitted](/docs/concepts#admission).
 
-### AdmissionCheck
+### API
 
-AdmissionCheck is a non-namespaced API object used to define details about an admission check like:
+AdmissionCheck is a non-namespaced API object used to define details about an admission check:
 
-- **controllerName** - It's an identifier for the controller that processes this AdmissionCheck, not necessarily a Kubernetes Pod or Deployment name. Cannot be empty.
-- **retryDelayMinutes** - Specifies how long to keep the workload suspended after a failed check (after it transitioned to False). After that the check state goes to "Unknown". The default is 15 min.
-- **parameters** - Identifies an additional resource providing additional parameters for the check.
+- **controllerName** - identifies the controller that processes the AdmissionCheck, not necessarily a Kubernetes Pod or Deployment name. Cannot be empty.
```

Comment:
Please also align the API comments, as they are user to generate the API reference: https://kueue.sigs.k8s.io/docs/reference/

Reply:
Fixed